app/views/comments/create.js.erb

```diff
@@ -1,17 +1,11 @@
 var message = "<%= escape_javascript(@message) %>";
-console.log(message);
-var responseText = '';
-$('#comment-form').on("ajax:success", function(e, data, status, xhr){
-  if (message == 'success') {
-    responseText = '';
-  } else if (message == 'invalid') {
-    responseText = 'Something is Wrong.';
-  } else {
-    responseText = 'Please try again later.';
-  }
-  $('#comments__response-text').val(responseText);
-  $('.comment-area').append("<%= j render @comment%>");
-}).on("ajax:error", function(e, data, status, xhr){
-  $('#handraisers__response-text').val('Please try again later.');
-});
-
+$("#comments__response-text<%= @comment.commentable_id %>").val("");

```

Comment:
What's the line doing? No value is being saved in any variable

Reply:
Removing the data of comment from form (with id = "comments__response-text<%= @comment.commentable_id %>")

Reply:
I don't think so, AFAICT its just returning value of jQuery element which is not being used anywhere.

Reply:
when I click on submit button it clears the comment's field

Reply:
Ok right!